### PR TITLE
mysql deprecation: add more troubleshooting steps

### PR DESCRIPTION
**Pull request by isacikgoz**
#### Summary
Add more troubleshooting steps to address a few issues pointed out from our community.

- A valid JSON issue happened in https://github.com/mattermost/migration-assist/issues/17
- An issue where some columns were missing in target database https://github.com/mattermost/migration-assist/issues/25
- An issue with pre-compiled `pgloader` https://github.com/mattermost/migration-assist/issues/29

### CODE REVIEW

File: source/deploy/postgres-migration.rst [hunk 1]
```diff
@@ -61,9 +61,55 @@ Errors during the pgloader command execution
...
+Failed to Find Column or Table
+::::::::::::::::::::::::::::::
+
+If you receive an error message similar to the following:
+
+.. code-block:: text
+
+   pgloader failed to find column
+
+That would mean that the column or table is missing in the Postgres database. You can fix this issue by checking whether you have created the correct version of Postgres schema. After re-creating the shcema, you can run the ``pgloader`` command again.
```
Comment (cwarnermm) [inline source/deploy/postgres-migration.rst:96]:
```suggestion
The column or table is missing in the PostgreSQL database. You can fix this issue by checking whether you have created the correct version of Postgres schema. After re-creating the schema, you can run the ``pgloader`` command again.
```

File: source/deploy/postgres-migration.rst [hunk 2]
```diff
@@ -61,9 +61,55 @@ Errors during the pgloader command execution
...
+
+Fell Through ECASE Expression
```
Comment (cwarnermm) [inline source/deploy/postgres-migration.rst:98]:
```suggestion
Fell through ECASE expression
```

File: source/deploy/postgres-migration.rst [hunk 3]
```diff
@@ -61,9 +61,55 @@ Errors during the pgloader command execution
...
+
+.. code-block:: text
+
+   ERROR mysql: 76 fell through ECASE expression.
+
+It is a `known issue <https://github.com/dimitri/pgloader/issues/1183>`_ with pgloader. You can fix this issue by either compiling ``pgloader`` from source or simply avoid this by running ``pgloader`` with our docker image. See: :ref:`install pgloader <deploy/manual-postgres-migration:install pgloader>` for more information.  
+
+.. note::
+
+  Also, there may be cases where ``pgloader`` continue to migrate remaining tables and skip one or more tables during migration. In such cases, it is recommended to identify issues with the table and fix them before running the ``pgloader`` command again with a clean database. It is possible to run the ``pgloader`` command with the ``--debug`` flag to get more information about the errors.
```
Comment (cwarnermm) [inline source/deploy/postgres-migration.rst:111]:
```suggestion
  Also, there may be cases where pgloader continues to migrate remaining tables and skip one or more tables during migration. In such cases, we recommend identifying issues with the table and fixing them before running the ``pgloader`` command again with a clean database. It is possible to run the ``pgloader`` command with the ``--debug`` flag to get more information about the errors.
```

**Review by cwarnermm — COMMENTED** [review]
Provided editorial and syntax feedback inline.

**Review by cwarnermm — APPROVED** [review]

**Review by agnivade — APPROVED** [review]

**Pull request merged**